<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QDialog" name="SettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>194</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>283</width>
    <height>194</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>283</width>
    <height>194</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>164</y>
     <width>166</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>133</y>
     <width>178</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>115</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Speed on/off of light:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="TE_SpeedOnOffLight">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>23</height>
       </size>
      </property>
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::MinuteSection</enum>
      </property>
      <property name="displayFormat">
       <string>mm:ss</string>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
      <property name="time">
       <time>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>102</y>
     <width>198</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>115</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Turn off the light after:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="TE_TurnOffLight">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>23</height>
       </size>
      </property>
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::MinuteSection</enum>
      </property>
      <property name="displayFormat">
       <string>mm:ss</string>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
      <property name="time">
       <time>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="RB_TurnOffLight">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>40</y>
     <width>198</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>115</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Maintain lux level:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="SB_MaintainLuxLevel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>23</height>
       </size>
      </property>
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>65536</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="RB_MaintainLuxLevel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>71</y>
     <width>178</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>115</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Maintain lux level step:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="SB_MaintainLuxLevelstep">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>23</height>
       </size>
      </property>
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>259</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>115</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>ON / OFF Time:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="TE_OnTime">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>23</height>
       </size>
      </property>
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::HourSection</enum>
      </property>
      <property name="displayFormat">
       <string>hh:mm</string>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
      <property name="time">
       <time>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="TE_OffTime">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>23</height>
       </size>
      </property>
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::HourSection</enum>
      </property>
      <property name="displayFormat">
       <string>hh:mm</string>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
      <property name="time">
       <time>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="RB_OnOffTime">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
